Tasks/DownloadGitHubReleaseV0/task.loc.json

```diff
@@ -10,7 +10,7 @@
   "demands": [],
   "version": {
     "Major": 0,
-    "Minor": 148,
+    "Minor": 156,
```

Comment:
this should be same as in **task.json** (auto generated on build)

Reply:
I did not rebuild the task before the final check in. Hence, this change. It would be 149 otherwise.